src/pseudopeople/schema_entities.py

```diff
@@ -395,6 +395,10 @@ class __Datasets(NamedTuple):
             COLUMNS.race_ethnicity,
         ),
         date_column="year",
+        row_noise_types=(
```

Comment:
I can't comment on line 369, but we should remove the default row noise types from Datasets and define them explicitly for each one. It made sense to have default values when we thought almost all the datasets would have the same exact row noise types, but that is not the case anymore.

Reply:
I've updated for this.